components/store/ProgramTag.js

```diff
@@ -0,0 +1,89 @@
+import { FontAwesome5 } from '@expo/vector-icons';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Chip } from 'react-native-paper';
+import Colors from '../../constants/Colors';
+import { styles } from '../../styled/store';
+import { Caption } from '../BaseComponents';
+
+/**
+ * @prop
+ * */
+
+// Capitalizes the first letter of every word in a phrase.
+// Note: catch special cases like certain words that are all uppercase.
+function capitalizeFirstLetters(word) {
+  // Separate each individual word in phrase.
+
+  // Special case since SNAP is all uppercase
+  if (word.toLowerCase() === 'snap match') {
+    return 'SNAP Match';
+  }
+
+  const splitWord = word.toLowerCase().split(' ');
+
+  for (var i = 0; i < splitWord.length; i++) {
+    splitWord[i] =
+      splitWord[i].charAt(0).toUpperCase() + splitWord[i].substring(1);
+  }
+
+  return splitWord.join(' ');
+}
+
+// program: a string representing the program name (valid programs are keys in programToIcon)
+function ProgramTag({ program }) {
+  let programLabel = program;
+
+  // Error checking: We'll fix the program label if it was somehow entered incorrectly
+  // i.e. "WIC" was entered as "WIc" or "SNAP Match" was entered as "snap Match"
+  // Hopefully ensures that chips will work most of the time, only not displaying if
+  // they enter the wrong string of words
+  if (
```

Comment:
Nice that you considered these cases, but wouldn't this input only come from within our code? Why would we need this here?

Reply:
I figured that the program name gets passed from the so many sources (i.e. store card, store details, store list and maybe more in the future) so if there's ever a case someone spells it wrong or something it would still work ?? it's a bit extra but i guess it ensures that the chips still function correctly even if there's a small capitalization error in the code

Reply:
i agree with this sentiment much human error